contracts/erc1155/contracts/src/ERC1155.sol

```diff
@@ -18,15 +18,13 @@
 
 pragma solidity ^0.5.5;
```

Comment:
If we're removing `SafeMath`, we should probably grep for `^0.5.5` and replace with `^0.5.9`.